things/postgres/channels.go

```diff
@@ -34,9 +35,13 @@ func NewChannelRepository(db *sqlx.DB) things.ChannelRepository {
 	}
 }
 
-func (cr channelRepository) Save(_ context.Context, channel things.Channel) (string, error) {
+func (cr channelRepository) Save(ctx context.Context, channel things.Channel) (string, error) {
+	span := opentracing.SpanFromContext(ctx)
+	defer span.Finish()
```

Comment:
Isn't this span already closed in `tracing` middleware from `things/tracing` package? Same goes for other spans.

Reply:
Thank you.  Removed extra span.Finish() calls.